index.html

```diff
@@ -8708,10 +8708,37 @@ <h2 id="AudioWorkletGlobalScope">
 [Global=(Worklet, AudioWorklet), Exposed=AudioWorklet]
 interface AudioWorkletGlobalScope : WorkletGlobalScope {
     void registerProcessor (DOMString name, VoidFunction processorCtor);
+    readonly attribute double currentTime;
+    readonly attribute float  sampleRate;
 };
           </pre>
           <section>
             <h3>
+              Attributes
+            </h3>
+            <dl class="attributes" data-dfn-for="AudioWorkletGlobalScope"
+            data-link-for="AudioWorkletGlobalScope">
+              <dt>
+                <dfn><code>currentTime</code></dfn> of type <span class=
+                "idlAttrType"><code>double</code></span>, readonly
+              </dt>
+              <dd>
+                The context time of the block of audio being processed. By
+                definition this will be equal to the value of
+                <a>BaseAudioContext</a>'s <a>currentTime</a> attribute that
```

Comment:
Does the link to currentTime actually work?

Reply:
@hoch I think you need the <a data-link-for="BaseAudioContext"> qualifier here